src/main/java/hw_2/Task_2.java

```diff
@@ -0,0 +1,20 @@
+package hw_2;
+
+public class Task_2 {
+    public static void main(String[] args) {
+        String string = "Testing, is my favourite job";
+        int a = string.indexOf(' ');
+        System.out.println(a);
+        String[] words = string.split(" ");
+        for (String val : words) {
+            System.out.println(val + " length " + val.length());
+        }
+        for (int i = 1; i < words.length; i++) {
+            if (a < words[i].length()) {
+                System.out.println(" true ");
+            }
+            System.out.println(" false");
+            break;
+        }
```

Comment:
This indexOf method return not first word length but index of first white space(" "). So the last cycle is incorrect. I'd recommend remove the first cycle and on the row 13(after first cycle) create a variable "firstWord" and get it from array(apply to it .replace(",", "")) to remove "," symbol; alse create a boolean result = true;
And then create a cycle where you will verify if the firstWord.length < words[i] length then result = false; break;

Reply:
fixed